app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_many :microposts, dependent: :destroy
```

Comment:
research dòng này
gợi ý: rails association

Reply:
has_many dùng để định nghĩa quan hệ của bảng User so với bảng Micropost là thuộc quan hệ one to many, tức là 1 user có thể tạo 1 hoặc nhiều micropost
dependent: :destroy dùng để khi mình xoá User thì các record của bảng Micropost liên quan đến User cũng sẽ bị xoá